esphome/components/shelly_dimmer/stm32flash.cpp

```diff
@@ -474,6 +474,19 @@ template<size_t N> void populate_buffer_with_address(uint8_t (&buffer)[N], uint3
   buffer[4] = static_cast<uint8_t>(buffer[0] ^ buffer[1] ^ buffer[2] ^ buffer[3]);
 }
 
+template<typename T> stm32_unique_ptr make_stm32_with_deletor(T ptr) {
+  static const auto CLOSE = [](stm32_t *stm32) {
+    if (stm32) {
+      free(stm32->cmd);  // NOLINT
+    }
+
+    free(stm32);  // NOLINT
```

Comment:
minor cosmic   - remove the empty line
```suggestion
    }
    free(stm32);  // NOLINT
```